presto-parser/src/main/java/com/facebook/presto/sql/ExpressionFormatter.java

```diff
@@ -553,6 +557,24 @@ static String formatSortItems(List<SortItem> sortItems, boolean unmangleNames)
                 .iterator());
     }
 
+    static String formatGroupBy(List<GroupBySpecification> groupBySpecifications)
+    {
+        for (GroupBySpecification groupBySpecification : groupBySpecifications) {
```

Comment:
Either produce correct result when `groupBySpecification` contains more than 1 elements, or assert that there is exactly one.